layouts/partials/navbar.html

```diff
@@ -1,33 +1,55 @@
-{{ $cover := .HasShortcode "blocks/cover" }}
-<nav class="js-navbar-scroll navbar navbar-expand navbar-dark {{ if $cover}} td-navbar-cover {{ end }} flex-column flex-md-row td-navbar" data-auto-burger="primary">
-        <a class="navbar-brand img-fluid" href="{{ .Site.Home.RelPermalink }}"></a>
-	<div class="td-navbar-nav-scroll ml-md-auto" id="main_navbar">
-		
-		<ul class="navbar-nav mt-2 mt-lg-0">
-			{{ $p := . }}
-			{{ $sections := slice "docs" "blog" "training" "partners" "community" "case-studies" }}
-			{{ range $sections }}
-			{{ with site.GetPage "section" . }}
-			<li class="nav-item mr-2 mb-lg-0">
-				{{ $active := eq .Section $.Section }}
-				<a class="nav-link{{if $active }} active{{end}}" href="{{ .RelPermalink }}" >{{ .Title }}</a>
-			</li>
-			{{ end }}
-			{{ end }}
-			{{ if  .Site.Params.versions }}
-			<li class="nav-item mr-n3 mr-lg-0 dropdown">
-				{{ partial "navbar-version-selector.html" . }}
-			</li>
-			{{ end }}
-			{{ if  (gt (len .Site.Home.Translations) 0) }}
-			<li class="nav-item mr-n4 mr-lg-0  dropdown">
-				{{ partial "navbar-lang-selector.html" . }}
-			</li>
-			{{ end }}
-			<li class="search-item nav-item mr-n4 mr-lg-0">
-				{{ partial "search-input.html" . }}
-			</li>
-		</ul>
-	</div>
-	<button id="hamburger" onclick="kub.toggleMenu()" data-auto-burger-exclude><div></div></button>
+{{ $cover := and
+    (.HasShortcode "blocks/cover")
+    (not .Site.Params.ui.navbar_translucent_over_cover_disable)
+-}}
+
+<nav class="js-navbar-scroll navbar navbar-expand navbar-dark
+            {{- if $cover }} td-navbar-cover {{- end }} flex-md-column flex-xl-row td-navbar" data-auto-burger="primary">
+  <a class="navbar-brand" href="{{ .Site.Home.RelPermalink }}">
+    {{- /**/ -}}
+    <span class="navbar-brand__logo navbar-logo">
+    </span>
+    {{- /**/ -}}
+    <span class="navbar-brand__name">
+      {{- .Site.Title -}}
+    </span>
+    {{- /**/ -}}
+  </a>
+  <div class="td-navbar-nav-scroll d-none d-md-block ml-xl-auto" id="main_navbar">
+    {{- /* In Docsy, the mt-2 class is used but that causes issues. This should be re-visited after 0.7 migration */ -}}
+    <ul class="navbar-nav mt-lg-0">
+      {{ $p := . -}}
+      {{ $sections := slice "docs" "blog" "training" "partners" "community" "case-studies" }}
```

Comment:
Eventually we want to get rid of this - see how https://github.com/kubernetes/website/pull/48363 does it.

I recommend at least adding a comment that it's a rough point, and ideally (maybe a separate PR) fetching the section list from querying English section pages' front matter  rather than hard-coding.

Reply:
The `menu` entry is on [`content/en/docs/home/_index.md`](https://github.com/kubernetes/website/blob/main/content/en/docs/home/_index.md?plain=1) because that's what's right for Docsy.

If we can copy the Docsy logic, either for all localizations or just for a subset that have opted in, that's ideal.

I'm more convinced now that the change belongs in a separate PR, but definitely a comment to explain it all is welcome.

Reply:
Alright, I will add a comment and create an issue to keep track of it.